tests/cnf/core/network/policy/tests/multinetpolicy_bondoversriov.go

```diff
@@ -0,0 +1,693 @@
+		Expect(err).ToNot(HaveOccurred(), "Sriov and MCP are not stable")
+
+		By("Deploy Test Resources: 2 NADs for bond CNI")
+		defineAndCreateBondNAD("bond", testNs1)
+		defineAndCreateBondNAD("bond", testNs2)
+
+		By("Deploy Test Resources: 5 Pods")
+		testPod1 = defineAndCreatePodWithBondIf(pod1, testNs1, ns1+nicPf1, ns1+nicPf2)
+		testPod2 = defineAndCreatePodWithBondIf(pod2, testNs1, ns1+nicPf1, ns1+nicPf2)
+		testPod3 = defineAndCreatePodWithBondIf(pod3, testNs1, ns1+nicPf1, ns1+nicPf2)
+		testPod4 = defineAndCreatePodWithBondIf(pod4, testNs2, ns2+nicPf1, ns2+nicPf2)
+		testPod5 = defineAndCreatePodWithBondIf(pod5, testNs2, ns2+nicPf1, ns2+nicPf2)
+
+		// Expected Result 1's and 0's represent Pass and Fail respectively when converted to Binary.
+		// Since we have 3 ports per destination pod, Octal notation is intuitive for readability.
+		// Example: expected Result (Octal) 0o70 --> (Binary) 0b111000 represents as below for the destination pod.
+		//
+		//   Pass     Pass     Pass     Fail     Fail     Fail
+		//     1       1        1        0         0       0
+		// tcp:5001 tcp:5002 udp:5003 tcp:5001 tcp:5002 udp:5003
+		// --IPv4-- --IPv4-- --IPv4-- --IPv6-- --IPv6-- --IPv6--
+		// -------------------Destination Pod-------------------
+		//
+		// Appropriate octal number is passed for each test.
+		By("Check egress traffic from pod1 to other 4 pods is working")
+		defineNmapValidPaths(
+			[]*pod.Builder{testPod1}, []*pod.Builder{testPod2, testPod3, testPod4, testPod5}, []int64{0o77, 0o77, 0o77, 0o77})
+
+		By("Check ingress traffic to pod1 from other 4 pods is working")
+		defineNmapValidPaths(
+			[]*pod.Builder{testPod2, testPod3, testPod4, testPod5}, []*pod.Builder{testPod1}, []int64{0o77, 0o77, 0o77, 0o77})
+	})
+
+	AfterEach(func() {
+		err := tNs1.CleanObjects(1*time.Minute, networkpolicy.GetMultiNetworkGVR())
+		Expect(err).ToNot(HaveOccurred(), "failed to clean Multi-NetworkPolicies in test namespace")
+	})
+
+	AfterAll(func() {
+		By("Removing SRIOV configuration and wait for MCP stable")
+		err := netenv.RemoveSriovConfigurationAndWaitForSriovAndMCPStable()
+		Expect(err).ToNot(HaveOccurred(), "Failed to remove SRIOV configuration and MCP stable")
+
+		By("Delete test namespace")
+		err = tNs1.Delete()
+		Expect(err).ToNot(HaveOccurred(), "Failed to delete test namespace")
+		err = tNs2.Delete()
+		Expect(err).ToNot(HaveOccurred(), "Failed to delete test namespace")
+	})
+
+	It("Egress - block all", reportxml.ID("77169"), func() {
+
+		By("Create Multi Network Policy")
+		_, err := networkpolicy.NewMultiNetworkPolicyBuilder(APIClient, "egress-deny", testNs1).
+			WithNetwork(fmt.Sprintf("%s/bond,%s/bond", testNs1, testNs2)).
+			WithPodSelector(metav1.LabelSelector{MatchLabels: map[string]string{"app": "pod1"}}).
+			WithPolicyType(multinetpolicyapiv1.PolicyTypeEgress).
+			Create()
+		Expect(err).ToNot(HaveOccurred(), "Failed to create Multi Network Policy")
+
+		By("Check egress traffic from pod1 to other 4 pods is working")
```

Comment:
Please fix the By or add a comment to show what exact do we expect

Reply:
Updated all `By` with relevant comments.